src/printers/jsx.js

```diff
@@ -77,4 +77,25 @@ export default {
   JSXText: ({node}) => node.value.trim()
     ? Tokens.string(node.value.trim())
     : null,
+
+  JSXMemberExpression: ({node, print}) => [
```

Comment:
can these be computed?

https://github.com/kyldvs/fjs/blob/master/src/printers/expressions.js#L165